packages/discord.js/src/structures/Poll.js

```diff
@@ -82,15 +111,55 @@ class Poll extends Base {
     } else {
       this.resultsFinalized ??= false;
     }
+
+    if (data.question) {
+      this.question = {
+        text: data.question.text,
+      };
+    }
+
+    if (data.answers) {
```

Comment:
Remove the `this.answers` update from the constructor in favor of this one. (You'll also need to move the initial collection set here)